nautobot/extras/forms/base_classes.py

```diff
@@ -0,0 +1,41 @@
+from nautobot.utilities.forms import BootstrapMixin
```

Comment:
I know it's pedantic but in keeping with common practices can we please just name this module `base.py`?